cli/args/mod.rs

```diff
@@ -1088,10 +1091,26 @@ impl CliOptions {
     &self,
     config_type: TsConfigType,
   ) -> Result<TsConfigForEmit, AnyError> {
-    deno_config::get_ts_config_for_emit(
+    let result = deno_config::get_ts_config_for_emit(
       config_type,
       self.maybe_config_file.as_ref(),
-    )
+    );
+
+    match result {
+      Ok(mut ts_config_for_emit) => {
+        if matches!(self.flags.subcommand, DenoSubcommand::Bundle(..)) {
+          // For backwards compatibility, force `experimentalDecorators` setting
+          // to true.
+          *ts_config_for_emit
+            .ts_config
+            .0
+            .get_mut("experimentalDecorators")
+            .unwrap() = serde_json::Value::Bool(true);
```

Comment:
Do we always have a `experimentalDecorators` field in `ts_config.0`? Might want to do:
```suggestion
            .entry("experimentalDecorators")
            .and_modify(|f| *f = serde_json::Value::Bool(true))
            .or_insert(serde_json::Value::Bool(true));
```

Reply:
Yes, this value is always filled by `deno_config`